src/Vex/Shaders/ShaderCompiler.cpp

```diff
@@ -42,9 +42,12 @@ ShaderEnvironment ShaderCompiler::CreateShaderEnvironment(ShaderCompilerBackend
     env.defines.emplace_back("VEX_SHIPPING", std::to_string(VEX_SHIPPING));
     env.defines.emplace_back("VEX_RAYTRACING",
                              GPhysicalDevice->featureChecker->IsFeatureSupported(Feature::RayTracing) ? "1" : "0");
-
-    rhi->ModifyShaderCompilerEnvironment(compiler, env);
-
+#if VEX_VULKAN
```

Comment:
We could possibly take advantage of this to change these macros to match the c++ ones, in that they are always defined, just with 1 or 0. (Instead of how it is currently where they aren't defined if not the right API).